video/out/drm_common.c

```diff
@@ -74,7 +74,10 @@ const struct m_sub_options drm_conf = {
     .opts = (const struct m_option[]) {
         {"drm-device", OPT_STRING(drm_device_path), .flags = M_OPT_FILE},
         {"drm-connector", OPT_STRING_VALIDATE(drm_connector_spec,
```

Comment:
Eliminating card number magic - yes please. Do note that the `drmDevices[]` order is unstable, so relying on that isn't great.

Hope I get to finish my `Add PATH_ID_TAG-like string to drmDevices` some time this week and push it to libdrm.